src/main/java/com/fabriccommunity/spookytime/common/SpookyItems.java

```diff
@@ -1,23 +1,33 @@
 package com.fabriccommunity.spookytime.common;
 
 import com.fabriccommunity.spookytime.SpookyTime;
+
+import net.minecraft.item.BlockItem;
 import net.minecraft.item.Item;
+import net.minecraft.item.ItemGroup;
 import net.minecraft.util.Identifier;
 import net.minecraft.util.registry.Registry;
 
 public class SpookyItems
 {
 
 
+    public static BlockItem TINY_PUMPKIN;
+
     public static void init() {
-        // NO-OP
+        register("tiny_pumpkin", TINY_PUMPKIN = new BlockItem(SpookyBlocks.TINY_PUMPKIN, new Item.Settings().group(ItemGroup.MISC)));
     }
 
     private static Item register(String name, Item item)
     {
         return Registry.register(Registry.ITEM, new Identifier(SpookyTime.MODID, name), item);
     }
 
+    private static Item register(String name, BlockItem item)
+    {
+        return Registry.register(Registry.ITEM, new Identifier(SpookyTime.MODID, name), item);
+    }
+
```

Comment:
I'm not sure why a new method is necessary? A BlockItem is still an Item so the previous register method should work just fine.